frontend/src/__tests__/cypress/cypress/e2e/userManagement/userManagement.cy.ts

```diff
@@ -0,0 +1,51 @@
+import { mockGroupSettings } from '~/__mocks__/mockGroupConfig';
+import { userManagement } from '~/__tests__/cypress/cypress/pages/userManagement';
+
+describe('User Management', () => {
+  beforeEach(() => {
+    cy.interceptOdh('GET /api/groups-config', mockGroupSettings());
+    cy.interceptOdh('PUT /api/groups-config', mockGroupSettings()).as('saveGroupSetting');
+    userManagement.visit();
+  });
+
+  it('Administrator group setting', () => {
+    const administratorGroupSection = userManagement.getAdministratorGroupSection();
+    administratorGroupSection.shouldHaveAdministratorGroupInfo();
+    administratorGroupSection.clearMultiChipItem();
+    administratorGroupSection.selectMultiGroup('odh-admins');
+    administratorGroupSection.findMultiSelectGroupInput().type('odh-admin');
+    administratorGroupSection.findMultiGroupOptions('odh-admins-1').click();
+    administratorGroupSection.findChipItem('odh-admins').should('exist');
+    administratorGroupSection.removeChipItem('odh-admins');
+    administratorGroupSection.findChipItem(/^odh-admins$/).should('not.exist');
+    administratorGroupSection.removeChipItem('odh-admins-1');
+    administratorGroupSection.findErrorText().should('exist');
+    administratorGroupSection.findMultiGroupOptions('odh-admins').click();
+    administratorGroupSection.findErrorText().should('not.exist');
+    userManagement.findSubmitButton().should('be.disabled');
+  });
+
+  it('User group setting', () => {
+    const userGroupSection = userManagement.getUserGroupSection();
+    userManagement.findSubmitButton().should('be.disabled');
+    userGroupSection.findErrorText().should('exist');
+    userGroupSection.selectMultiGroup('odh-admins');
+    userGroupSection.findMultiGroupOptions('system:authenticated').click();
+    userGroupSection.findChipItem('odh-admins').should('exist');
+    userGroupSection.removeChipItem('odh-admins');
+    userManagement.findSubmitButton().should('be.enabled');
+    userManagement.findSubmitButton().click();
+    cy.wait('@saveGroupSetting').then((interception) => {
```

Comment:
Is it possible to check whether the alert is visible after this `@saveGroupSetting` is successfully executed? I believe there should be something to check whether this operation is completed. Correct me if I'm wrong.